examples/src/main/java/io/dapr/examples/pubsub/bulk/README.md

```diff
@@ -0,0 +1,314 @@
+name: Run Publisher
+match_order: sequential
+expected_stdout_lines:
+  - '== APP == Published the set of messages in a single call to Dapr'
+  - '== APP == Done'
+background: true
+sleep: 15
+-->
+
+```bash
+dapr run --components-path ./components/pubsub --app-id bulk-publisher -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.pubsub.bulk.BulkPublisher
+```
+
+<!-- END_STEP -->
+
+Once running, the Publisher should print the output as follows:
+
+```txt
+✅  You're up and running! Both Dapr and your app logs will appear here.
+
+== APP == Using preview client...
+== APP == Constructing the list of messages to publish
+== APP == Going to publish message : This is message #0
+== APP == Going to publish message : This is message #1
+== APP == Going to publish message : This is message #2
+== APP == Going to publish message : This is message #3
+== APP == Going to publish message : This is message #4
+== APP == Going to publish message : This is message #5
+== APP == Going to publish message : This is message #6
+== APP == Going to publish message : This is message #7
+== APP == Going to publish message : This is message #8
+== APP == Going to publish message : This is message #9
+== APP == Done
+
+```
+
+Messages have been published in the topic.
+
+Once running, the Subscriber should print the output as follows:
+
+```txt
+== APP == Subscriber got: This is message #0
+== APP == Subscriber got: {"id":"bb2f4833-0473-446b-a6cc-04a36de5ac0a","source":"bulk-publisher","type":"com.dapr.event.sent","specversion":"1.0","datacontenttype":"text/plain","data":"This is message #0","data_base64":null}
+== APP == Subscriber got: This is message #1
+== APP == Subscriber got: {"id":"323935ed-d8db-4ea2-ba28-52352b1d1b34","source":"bulk-publisher","type":"com.dapr.event.sent","specversion":"1.0","datacontenttype":"text/plain","data":"This is message #1","data_base64":null}
+== APP == Subscriber got: This is message #2
+== APP == Subscriber got: {"id":"2976f254-7859-449e-b66c-57fab4a72aef","source":"bulk-publisher","type":"com.dapr.event.sent","specversion":"1.0","datacontenttype":"text/plain","data":"This is message #2","data_base64":null}
+== APP == Subscriber got: This is message #3
+== APP == Subscriber got: {"id":"f21ff2b5-4842-481d-9a96-e4c299d1c463","source":"bulk-publisher","type":"com.dapr.event.sent","specversion":"1.0","datacontenttype":"text/plain","data":"This is message #3","data_base64":null}
+== APP == Subscriber got: This is message #4
+== APP == Subscriber got: {"id":"4bf50438-e576-4f5f-bb40-bd31c716ad02","source":"bulk-publisher","type":"com.dapr.event.sent","specversion":"1.0","datacontenttype":"text/plain","data":"This is message #4","data_base64":null}
+== APP == Subscriber got: This is message #5
+== APP == Subscriber got: {"id":"07bad175-4be4-4beb-a983-4def2eba5768","source":"bulk-publisher","type":"com.dapr.event.sent","specversion":"1.0","datacontenttype":"text/plain","data":"This is message #5","data_base64":null}
```

Comment:
You cannot use UUID since it varies on every run. Unless you use a generator that is deterministic.

Reply:
Here I am not validating that it got a particular message, I am only validating that it got the message i.e. `Subscriber got: This is message #5`. I am not validating the ID.

Reply:
OK.